### PR TITLE
docs: cleanup 03_celery.md

### PR DESCRIPTION
**Pull request by ardieorden**
- Fixes a typo with the import and variable names
- Adds a missing url

Thanks for the tutorial!

### CODE REVIEW

File: docs/03-celery.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ def dummy_task():
 To check it works, let's call it directly using the Python REPL (`python`):
 
 ```python
->>> import fastapi_celery.task
+>>> import fastapi_celery.task as task
```
Comment (derlin) [inline docs/03-celery.md]: I would suggest the following instead:
```
from fastapi_celery import task
```

**Comment by derlin** [comment]
Thank you for taking the time to leave a feedback, I appreciate it!
Just a small remark and we should be good to go :)

**Comment by ardieorden** [comment]
Just finished applying the suggestion. And no problem.

**Comment by derlin** [comment]
Great! I will merge it as is. Next time, I would rather you fix the commit instead of adding a new one on top (with fixup commit+squash or interactive rebase). This makes the git history clearer :)

**Pull request merged**